MaterialLib/FractureModels/CreateCohesiveZoneModeI.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateCohesiveZoneModeI.h"
+
+#include "CohesiveZoneModeI.h"
+#include "ProcessLib/Utils/ProcessUtils.h"  // required for findParameter
+
+namespace MaterialLib
+{
+namespace Fracture
+{
+namespace CohesiveZoneModeI
+{
```

Comment:
The cohesive zone delamination model for mode I fracture introduced herein follows a bilinear traction-separation law which is characterized by four basic parameters, that is, the initial normal and shear stiffness K_n, K_s, the fracture toughness G_c (also referred to as critical energy release rate), and the peak tensile normal traction t_{n,p}.